### PR TITLE
80/investor dashboard

### PR DESCRIPTION
**Pull request by MatiasOS**
This a work in progress for #80 

- Fix mising `,` in `tslint.json`

### CODE REVIEW

File: src/provider/ethplorer.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import fetch from 'isomorphic-fetch';
+import { TokenInfo, Holder } from '../types';
+
+const ETHPLORER_API_URL =
+    process.env.REACT_APP_ETHPLORER_API_URL || 'https://api.ethplorer.io';
+const ETHPLORER_API_KEY = process.env.REACT_APP_ETHPLORER_API_KEY;
+const DXDAO_ADDRESS = '0xa1d65E8fB6e87b60FECCBc582F7f97804B725521';
```
Comment (AugustoL) [inline src/provider/ethplorer.ts:7]: you can get the address from the config store, see the rest of the app how we get it.

File: src/stores/Ethplorer.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    rootStore: RootStore;
+
+    constructor(rootStore) {
+        this.rootStore = rootStore;
+        this.getTokenInfo();
+        this.getHolders(20);
+    }
+
+    private async getTokenInfo(): Promise<void> {
+        this.tokenInfo = await getTokenInfo();
```
Comment (AugustoL) [inline src/stores/Ethplorer.ts:23]: maybe you can just get the address of the token directly form the config store here.

**Comment by AugustoL** [comment]
@MatiasOS to go to the dashboard please add a link that says "Investor Dashboard" at the left of the account when the account is connected.

**Comment by MatiasOS** [comment]
Sry team, I'm very busy with my current job and it's impossible to me keep working on this.

**Pull request closed without merging**